provisioner/windows-restart/provisioner.go

```diff
@@ -114,6 +114,12 @@ var waitForRestart = func(p *Provisioner, comm packer.Communicator) error {
 	var cmd *packer.RemoteCmd
 	trycommand := TryCheckReboot
 	abortcommand := AbortReboot
+
+	// This sleep works around an azure/winrm bug. For more info see
+	// https://github.com/hashicorp/packer/issues/5257; we can remove the
+	// sleep when the underlying bug has been resolved.
+	time.Sleep(1 * time.Second)
```

Comment:
this seems like the right direction, but my concerns are

* This only impacts ntlmssp connections. should we limit it to those?
* is 1 second long enough? What happens if it's not? Should we make it configurable?

Reply:
Is there an easy way to test for ntlmssp connections from inside the provisioner?  If so, that's definitely a good idea.  And in that case I'd be more willing to bump the sleep length "just in case"; I wanted to keep it short in order to not unduly impact other users.  For what it's worth, this actually works on my computer with a sleep of 500ms.

Reply:
> Is there an easy way to test for ntlmssp connections from inside the provisioner?

probably not, thinking about it...